kong.yml

```diff
@@ -94,10 +94,19 @@
 ######
 ## PostgreSQL configuration
 # postgres:
-#    host: "127.0.0.1"
-#    port: 5432
-#    user: kong
-#    database: kong
+  #####
+  ## Connection host and port settings
+  # host: "127.0.0.1"
+  # port: 5432
+
+  ######
+  ## Name of the database used by Kong. Will be created if it does not exist.
+  # database: kong
+
+  #####
+  ## User authentication settings
+  # user: ""
+  # password: ""
```

Comment:
Just keeping it consistent with the `cassandra` doc (https://github.com/Mashape/kong/blob/master/kong.yml#L97-L100). 

We agreed a while ago to keep the description on the website (configuration reference) and in `kong.yml` identical, so I am following that rule. If we want to change it, then it should change for every section to keep it consistent.